saleor/static/dashboard-next/theme.ts

```diff
@@ -12,7 +12,13 @@ const createShadow = (pv, pb, ps, uv, ub, us, av, ab, as) =>
 export const ICONBUTTON_SIZE = 48;
 
 export type IThemeColors = Record<
-  "primary" | "secondary" | "error" | "paperBorder" | "autofill",
+  | "primary"
+  | "secondary"
+  | "error"
+  | "paperBorder"
+  | "autofill"
+  | "buttonText"
+  | "textColor",
```

Comment:
`buttonText` and `textColor` belong to `Record<"default" | "gray", string>` type.